zulip/integrations/log2zulip/log2zulip

```diff
@@ -67,47 +66,53 @@ def process_lines(raw_lines, file_name):
 def process_logs():
     # type: () -> None
     for filename in log_files:
-        data_file_path = "/var/tmp/log2zulip.state"
+        data_file_path = os.path.join(tempfile.gettempdir(), 'log2zulip.state')
```

Comment:
So, there's a reason we were using `/var/tmp`, rather than `/tmp`, which is that `/tmp` gets cleared on boot (which would meant the bot would lose its state and thus repeat messages or something if the server reboots)!  

So I think we might just want it to accept a path to its state directory (to support Windows) and have the default remain as-is.